.chloggen/awss3receiver_notifications.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'new_component'
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: awss3receiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: 'Add support for monitoring the progress of ingesting data from an S3 bucket via OpAMP custom messages.'
```

Comment:
wow. That's the first time I see this done. Can you use obsreport to report metrics first?